cache/remotecache/azblob/utils.go

```diff
@@ -50,7 +51,13 @@ func getConfig(attrs map[string]string) (*Config, error) {
 		return &Config{}, errors.Wrap(err, "azure storage account url provided is not a valid url")
 	}
 
-	accountName := strings.Split(accountURL.Hostname(), ".")[0]
+	accountName, ok := attrs[attrAccountName]
```

Comment:
wondering if we should return some error if account name is still empty at the end of this

Reply:
sounds good